development/development/css/css_naming.rst

```diff
@@ -143,20 +142,20 @@ Further Reading
 JavaScript Hooks
 ~~~~~~~~~~~~~~~~
 
-As a rule, it is unwise to bind your CSS and your JS onto the same class in your HTML. This is because doing so means you can’t have (or remove) one without (removing) the other. It is much cleaner, much more transparent, and much more maintainable to bind your JS onto specific classes. Typically, this is why you sometimes se classes that are prepended with ``js-``, for
+As a rule, it is unwise to bind your CSS and your JS onto the same class in your HTML. This is because doing so means you can’t have (or remove) one without (removing) the other. It is much cleaner, much more transparent, and much more maintainable to bind your JS onto data attributes.
 
 example:
 
 .. code:: html
 
-    <input type="submit" class="btn js-btn" value="Follow" />
+    <input type="submit" class="btn" data-execute="addSomething" value="Follow" />
 
-This means that we can have an element elsewhere which can carry the style of ``.btn {}``, but without the behavior of ``.js-btn``.
+This means that we can have an element elsewhere which can carry the style of ``.btn {}``, but without the behavior of ``data-execute="addSomething"``.
 
 ``data-*`` Attributes
 ^^^^^^^^^^^^^^^^^^^^^
 
-A cleaner and preferred practice is to use ``data-*`` attributes as JS hooks. ``data-*`` attributes, as per the spec, are typically used to store custom data private to the page or application’. however since you are already binding this attribute to your js, it makes since to use the same attribute as the js hook.
+It is our preferred practice to use ``data-*`` attributes as JS hooks. ``data-*`` attributes, as per the spec, are typically used to store custom data private to the page or application’. however since you are already binding this attribute to your js, it makes since to use the same attribute as the js hook.
```

Comment:
```suggestion
It is our preferred practice to use ``data-*`` attributes as JS hooks. ``data-*`` attributes, as per the spec, are typically used to store custom data private to the page or application’. however since you are already binding this attribute to your js, it makes sense to use the same attribute as the js hook.
```